app/views/user_mailer/reset_password_instructions.zh-cn.html.erb

```diff
@@ -0,0 +1,7 @@
+<p><%= @resource.email %> ，嗨呀!!</p>
+
+<p>有人（但愿是你 ~ ）请求更改你 Mastodon 账户的密码。如果是你的话，请点击下面的链接：</p>
```

Comment:
Please, no tildes, no ASCII tildes, and no spaces around tildes or before a closing parenthesis.